data/mods/MindOverMatter/recipes/rituals.json

```diff
@@ -1,4 +1,20 @@
 [
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "name": "engineer mind",
+    "id": "recipe_mental_engineering",
+    "description": "With the help of a telepathic focusing tool and your (hopefully) extensive telepathic powers, you can sink into a deep trance where you can only sense yourself, allowing you to permanently adjust your mental state and view of the outside world.  Will the end result still be you?",
+    "category": "CC_PSIONIC",
+    "subcategory": "CSC_PSIONIC_RITUALS",
+    "skill_used": "metaphysics",
+    "difficulty": 9,
+    "time": "0 h",
+    "book_learn": [ [ "schematics_telepathic_focusing_tool", 4 ] ],
```

Comment:
Made a new report specific to this process and put the recipe in it.  Referenced above, but I imagined this tool as a general telepathic tool rather than a 'PATIENT MOSAIC' tool.